lib/module.js

```diff
@@ -619,7 +619,14 @@ Module._initPaths = function() {
     homeDir = process.env.HOME;
   }
 
-  var paths = [path.resolve(process.execPath, '..', '..', 'lib', 'node')];
+  var prefixDir;
+  if (isWindows) {
+    prefixDir = path.resolve(process.execPath, '..');
+  } else {
+    prefixDir = path.resolve(process.execPath, '..', '..');
+  }
+
+  var paths = [path.resolve(prefixDir, 'lib', 'node')];
```

Comment:
If I understand this correctly, on non-Windows we have

```
$PREFIX/
   bin/node
   lib/
```

and on Windows we have

```
$PREFIX\
  node.exe
  lib/
```

So the previous code was adding a path like `c:\Program Files\lib` to the default node search path.. which is just flat-out wrong?

I think the code above could benefit from a comment describing what its doing. Particularly if my guess above is wrong!